doc/musig-spec.mediawiki

```diff
@@ -212,8 +214,20 @@ Input:
 
 ==== Nonce Generation ====
 
-'''''NonceGen()''''':
-* Generate two random integers ''k<sub>1</sub>, k<sub>2</sub>'' in the range ''1...n-1''
+Input:
+* The secret signing key ''sk'': a 32-byte array or 0-byte array (optional argument)
+* The aggregate public key ''aggpk'': a 32-byte array or 0-byte array (optional argument)
+* The message ''m'': a 32-byte array or 0-byte array (optional argument)
+* The length ''l'' of auxiliary entropy ''ent'': an integer ''&ge; 0''
+* The auxiliary entropy ''ent'': an l-byte array (optional argument)
```

Comment:
If your pseudocode language is expressive compute `len(sk)`, couldn't it also compute `len(ent)` then?

Reply:
I didn't do this for consistency with other variable length arguments (number of pubkeys, nonces, etc) but I reconsidered and `len(ent)` is quite a bit nicer.